R/pkg/R/serialize.R

```diff
@@ -83,6 +83,7 @@ writeObject <- function(con, object, writeType = TRUE) {
          Date = writeDate(con, object),
          POSIXlt = writeTime(con, object),
          POSIXct = writeTime(con, object),
+         bigint = writeDouble(con, object),
```

Comment:
If R doesn't have `bigint` type, we should remove all `bigint` related logic. I don't know the history of `bigint` mapping in the Types.R file. Why should we have it since every big number is numeric (Double in the backend)?